### PR TITLE
Signed-off-by: DustDFG <dfgdust@gmail.com>

### PR DESCRIPTION
**Pull request by dustdfg**
I created new optimezed logos
Signed-off-by: DustDFG <dfgdust@gmail.com>

### CODE REVIEW

**Comment by biodranik** [comment]
What problem are you trying to solve? Also, if you moved old logos, their path should be also updated in the site code.

**Comment by dustdfg** [comment]
@biodranik Much unused metadata

**Comment by biodranik** [comment]
But why PNG?

**Comment by dustdfg** [comment]
Is only SVG enough?

**Comment by biodranik** [comment]
> Is only SVG enough?

PNGs are not used on the site. But they probably may be shared with someone.

**Comment by dustdfg** [comment]
@biodranik And now? I updated path to logo

File: sass/main.scss [hunk 1]
```diff
@@ -123,7 +123,7 @@ footer {
 }
 
 .logo {
-  background-image: url("/logo-on-white.svg");
+  background-image: url("/logos/vector/green-on-white.svg");
```
Comment (biodranik) [inline sass/main.scss:126]: It should be green-on-transparent. Did you check the site locally? Install [zola](https://www.getzola.org/) and run `zola serve` in the root folder.

**Comment by dustdfg** [comment]
@biodranik Hello, it seems to me that I fix my mistake

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**